build.make

```diff
@@ -20,7 +20,7 @@
 
 # This is the default. It can be overridden in the main Makefile after
 # including build.make.
-REGISTRY_NAME=quay.io/k8scsi
+REGISTRY_NAME?=quay.io/k8scsi
```

Comment:
This I understand: you want to do `REGISTRY_NAME=foo make push` instead of `make push REGISTRY_NAME=foo`.

Reply:
correct